src/Hubbup.Web/Client/Standup/components.tsx

```diff
@@ -0,0 +1,370 @@
+                    {content}
+                </div>;
+            }
+        }
+        else {
+            content = <div className="alert alert-info">{this.props.emptyText}</div>;
+        }
+
+        return <div>
+            <h4>
+                {this.props.title}
+                {' ' /* https://facebook.github.io/react/blog/2014/02/20/react-v0.9.html#jsx-whitespace */}
+                <span className="badge">{this.props.issues.length}</span>
+                {' ' /* https://facebook.github.io/react/blog/2014/02/20/react-v0.9.html#jsx-whitespace */}
+                {collapser}
+            </h4>
+            {content}
+        </div>;
+    }
+}
+
+type RateLimitUpdater = (graphQl: { cost: number, remaining: number, resetAt: string }, rest: { remaining: number, reset: string }) => void;
+
+interface PersonProps {
+    baseUrl: string,
+    repoSet: string,
+    login: string,
+    environment: string,
+    updateRateLimit: RateLimitUpdater;
+}
+
+class Person extends React.Component<PersonProps, { loading: boolean, error: string, data: Data.RepoSetIssueResult }> {
+    constructor(props: PersonProps) {
+        super(props);
+        this.state = {
+            loading: true,
+            error: "",
+            data: null
+        };
+    }
+
+    componentDidMount() {
+        // Can't await this here, but that's OK, it will call setState eventually.
+        this.reload();
+    }
+
+    async reload() {
+        // This doesn't immediately clear the loading widget, but it will when the function returns and we re-render.
+        this.setState({
+            loading: true,
+            error: "",
+            data: null
+        });
+
+        let resp = await fetch(`${this.props.baseUrl}api/repoSets/${this.props.repoSet}/issues/${this.props.login}`, fetchSettings)
+        if (resp.status < 200 || resp.status > 299) {
+            this.setState({
+                loading: false,
+                error: `Unexpected response fetching issues: ${resp.status} ${resp.statusText}`,
+                data: null
+            });
+            return;
+        }
+        else {
+            let data = (await resp.json()) as Data.RepoSetIssueResult;
+
+            // Update rate limit info
+            this.props.updateRateLimit(data.graphQlRateLimit, data.restRateLimit);
+
+            // Update my state
+            this.setState({
+                loading: false,
+                error: "",
+                data: data
+            });
+        }
+    }
+
+    render() {
+        let content;
+        let rateLimitDisplay;
+
+        let headerBadgeStyles = {
+            "marginLeft": "0.5em",
+            "marginTop": "-5px"
+        };
+
+        if (this.state.loading) {
+            content = <div className="panel-body">
+                <LoadingPanel />
+            </div>;
+        }
+        else if (this.state.error) {
```

Comment:
More nitpicking :) The brace style is weird here. Normally you'd see `} else if (...) {` all on one line.